NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export(add_ws_to_parse_data)
```

Comment:
what about the other internal functions that are exported now? Should I remove the export tag there as well?